### PR TITLE
docs MVP

### PR DESCRIPTION
**Pull request by mhausenblas**
This is the MVP for #10

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (sh0rez) [inline .gitignore:1]: Side-note: `.DS_Store` is a macOS specific thing. For OS / editor specific things global gitignore (https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) is often a nice solution.

Suggestions on how we treat such in the future? Add those to repo gitignore or request contributors to setup their own gitignore?

File: docs/content/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Welcome!
+
+Polly is a specification for parameterized packages of observability-related
+configuration objects such as dashboards, alerts and [more][targets].
+
+Wit polly packages we intend to address the fundamental challenges involved
```
Comment (sh0rez) [inline docs/content/index.md:6]:
```suggestion
With polly packages we intend to address the fundamental challenges involved
```

**Review by sh0rez — APPROVED** [review]
Cool!

File: .gitignore (hunk 1 above)
Comment (mhausenblas) [reply .gitignore:1]: I wouldn't overthink this. You're ofc right with your observation and in this phase I'm mainly focused on making it easy for folks to contribute, not establish too many barriers. If you feel strongly about it, I suggest you create an issue and let's discuss it there?

Comment (sh0rez) [reply .gitignore:1]: No need, just wanted to raise it in case it was an oversight. It totally doesn't matter in this stage of the project :+1:

**Pull request merged**